### ISSUE TITLE
Order Notional amounts

### ISSUE THREAD

**Issue by TheBigBeaver** [yogat3ch/AlpacaforR#46]
Do you know whether the order_submit function handles notional orders?
```
#> [1] TRUE
if (.open) {
  # if the market is open then place a market buy order for "BYND"
(bo <- order_submit("bynd", side = "b", q = 2, type = "m"))
}
```
In the `order_submit` doc I only see `qty(integer) Ordered quantity`

Although the Alpaca documentation mentions it is possible to pass notional orders

`notional` | string<number> | `Required` | dollar amount to trade - Cannot work with qty. Can only work for market order types and day for time in force.

**Comment by yogat3ch** [comment]
Hi @TheBigBeaver ,
I haven't implemented support for notional orders, that's new to me. Seems like it wouldn't be hard to implement though. 
Do you want to submit a PR for it?

**Comment by yogat3ch** [comment]
Hi @TheBigBeaver,
I just had a chance to implement notional order amounts. Cheers! 🎉

**Comment by yogat3ch** [comment]
Closed via 5b954e17227b77e35817895b940fc50de1fe6cb5
